### PR TITLE
Code style fixes

### PR DESCRIPTION
**Pull request by mmachado95**

### CODE REVIEW

**Comment by pedrocaseiro** [comment]
As long as pylint runs with no warnings, it's good to go.

File: dotworks/dotworks_server/views/views.py [hunk 1]
```diff
@@ -114,107 +112,122 @@ def register_action(request):
...
+                birth_date=birth_date,
+                degree=degree
+            )
             student.save()
-            user = authenticate(username = email, password = password)
+            user = authenticate(username=email, password=password)
             if user is not None:
                 login(request, user)
-                template = loader.get_template('dotworksServer/home.html')
                 return HttpResponse(template.render(context, request))
```
Comment (Joaquim-Ferrer) [inline dotworks/dotworks_server/views/views.py:138]: What is the template rendered if the previous line was erased?

File: dotworks/dotworks_server/views/views.py [hunk 2]
```diff
@@ -114,107 +112,122 @@ def register_action(request):
...
+            )
             internship.save()
     return HttpResponseRedirect(reverse('index'))
 
+
 @login_required(login_url=reverse_lazy('no_permission_error'))
 def internship_details(request, internship_id):
-    internship = Internship.objects.get(pk = internship_id)
-    template = loader.get_template('dotworksServer/internship_details.html')
+    internship = Internship.objects.get(pk=internship_id)
```
Comment (Joaquim-Ferrer) [inline dotworks/dotworks_server/views/views.py:187]: In this case there can be whitespaces between the '='? What is the criteria?

**Review by Joaquim-Ferrer — COMMENTED** [review]
Bottom line: Good to go if the register_action view still works.

**Review by Joaquim-Ferrer — APPROVED** [review]

**Pull request merged**